api/v1/postgres_types.go

```diff
@@ -481,3 +493,138 @@ func deleteIfEmpty(json map[string]interface{}, key string) {
 func init() {
 	SchemeBuilder.Register(&Postgres{}, &PostgresList{})
 }
+
+func (p *Postgres) buildAdditionalVolumes(c *corev1.ConfigMap) []zalando.AdditionalVolume {
+	if c == nil {
+		// abort if the global configmap is not there
+		return nil
+	}
+
+	return []zalando.AdditionalVolume{
+		{
+			Name:      "empty",
+			MountPath: "/opt/empty",
+			TargetContainers: []string{
+				"all",
+			},
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+		{
+			Name:      "postgres-exporter-configmap",
+			MountPath: "/metrics",
+			TargetContainers: []string{
+				ExporterSidecarName,
+			},
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: SidecarsCMName,
+					},
+					Items: []corev1.KeyToPath{
+						{
+							Key:  SidecarsCMExporterQueriesKey,
+							Path: "queries.yaml",
+						},
+					},
+				},
+			},
+		},
+		{
+			Name:      "postgres-fluentbit-configmap",
+			MountPath: "/fluent-bit/etc",
+			TargetContainers: []string{
+				FluentBitSidecarName,
+			},
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: SidecarsCMName,
+					},
+					Items: []corev1.KeyToPath{
+						{
+							Key:  SidecarsCMFluentBitConfKey,
+							Path: "fluent-bit.conf",
```

Comment:
Same as above. Could we use `SidecarsCMFluentBitConfKey`?

Reply:
Same as above :-)